script/check_jdk_version.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
We have a cmd/util directory where we should host any tool. Please, keep the same structure.

Reply:
ok